packages/fcl-wc/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @onflow/fcl-wc
 
+## 4.0.0
+
+### Patch Changes
+
+- Updated dependencies [[`3c99c856`](https://github.com/onflow/fcl-js/commit/3c99c8560f61b5b38238cbd0e93814936aee282e)]:
+  - @onflow/fcl@1.6.0
+
```

Comment:
This must be a bug in the changelogs CLI re. peer dependencies.  Will try to see if new version exists.